src/commands/releases.rs

```diff
@@ -197,10 +206,11 @@ fn execute_files_upload_sourcemaps<'a>(matches: &ArgMatches<'a>, config: &Config
     let paths = matches.values_of("paths").unwrap();
     let extensions = match matches.values_of("extensions") {
         Some(matches) => matches.map(|ext| OsStr::new(ext.trim_left_matches("."))).collect(),
-        None => vec![OsStr::new("js"), OsStr::new("map")],
+        None => vec![OsStr::new("js"), OsStr::new("map"), OsStr::new("jsbundle")],
```

Comment:
If react native runs in debug, it generates a file called
`index.ios.bundle`
Maybe we should also add `bundle` here